<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>640</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>6</width>
      <height>6</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="Map_Tab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>-1</y>
        <width>981</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QPushButton" name="marker_placeHolder8">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);
</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="marker_placeHolder12">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>12</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="marker_placeHolder5">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="marker_placeHolder2">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);
</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="marker_placeHolder4">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="marker_placeHolder9">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="marker_placeHolder6">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="marker_placeHolder3">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="marker_placeHolder7">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="marker_placeHolder10">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="marker_placeHolder1">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="marker_placeHolder11">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string>11</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="Image">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>-11</y>
        <width>1011</width>
        <height>641</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Pictures/Screenshots/HuntMap for Nea.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Edit_Setup_Button">
      <property name="geometry">
       <rect>
        <x>885</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Setup</string>
      </property>
     </widget>
     <zorder>Image</zorder>
     <zorder>gridLayoutWidget_2</zorder>
     <zorder>Edit_Setup_Button</zorder>
    </widget>
    <widget class="QWidget" name="Statistics_Tab">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="graphsGrrid">
       <item row="3" column="0">
        <widget class="QGroupBox" name="pie_chart_preview">
         <property name="title">
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="line_graph_preview">
         <property name="title">
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="bar_graph_preview">
         <property name="title">
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="pop_graph_preview">
         <property name="title">
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Prevention_Tab">
     <attribute name="title">
      <string>Prevention</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionPrevention">
   <property name="text">
    <string>Prevention</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>Tabs</tabstop>
  <tabstop>marker_placeHolder1</tabstop>
 </tabstops>
 <resources>
  <include location="../../Wireframes/NEA resources.qrc"/>
 </resources>
 <connections/>
</ui>
